jest.config.js

```diff
@@ -32,12 +34,5 @@ module.exports = {
       statements: 100,
     },
   },
-  coveragePathIgnorePatterns: ["/node_modules/", "<rootDir>/dist"],
-  testPathIgnorePatterns: [
-    "/node_modules/",
-    // By default we only run unit tests:
-    "/e2e/node/",
-    "/e2e/browser/",
```

Comment:
This depends on the `e2e` tests moving out of `src` right ?

Reply:
By default, jest would pick up anything that has a `.test` suffix, but having the e2e in their own directory makes it easier to have a simple matcher, yes.